libs/designer/src/lib/core/actions/bjsworkflow/settings.ts

```diff
@@ -877,16 +873,16 @@ const isInvokerConnectionSupported = (isTrigger: boolean, nodeType: string, root
 };
 
 const invokerConnection = (
-  connectionReferences: ConnectionReferences | undefined,
-  dispatch: Dispatch | undefined
+  isTrigger: boolean,
+  nodeType: string,
+  rootNodeId: string | undefined,
+  connectorId: string
 ): SimpleSetting<ConnectionReferences> | undefined => {
-  if (connectionReferences !== undefined) {
-    dispatch?.(addInvokerSupport({ connectionReferences }));
+  if (
+    isInvokerConnectionSupported(isTrigger, nodeType, rootNodeId) &&
+    connectorId === Constants.INVOKER_CONNECTION.DATAVERSE_CONNECTOR_ID
```

Comment:
Please do not add arm connectorid checks in the code. Add metadata in the manifest and manifest metadatas can be looked at but not connectorIds, we do not want such checks in the client code.
You can patch this connectors manifest in your client service to provide appropriate settings, if this is manifest based connector please patch it in your client service to add proper setting metadata for invokerconnection support.

Reply:
If there was a previous PR which checked in such constants, it will be removed because we do not want to follow this kind of model which was done in old designer.

Reply:
Removed the check for `connectorId`